metagpt/tools/tool_types.py

```diff
@@ -1,68 +1,80 @@
+from enum import Enum
+
 from metagpt.prompts.tool_types import (
     DATA_PREPROCESS_PROMPT,
     FEATURE_ENGINEERING_PROMPT,
     IMAGE2WEBPAGE_PROMPT,
     MODEL_EVALUATE_PROMPT,
     MODEL_TRAIN_PROMPT,
 )
-from metagpt.tools.tool_data_type import ToolType, ToolTypeEnum
-from metagpt.tools.tool_registry import register_tool_type
+from metagpt.tools.tool_data_type import ToolType
+
+Eda = ToolType(name="eda", desc="For performing exploratory data analysis")
 
+DataPreprocess = ToolType(
```

Comment:
It's a global variable, and the spec is an underscore like A_B, not a hump

Reply:
I try to avoid self assignment DATA_PREPROCESS = DATA_PREPROCESS in the enum